06-kfreebsd.diff by Petr Salinger <Petr.Salinger@seznam.cz>

Fixing FTBFS on GNU/kFreeBSD (Closes: #504820).

---
 progs/tset.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/progs/tset.c
+++ b/progs/tset.c
@@ -981,7 +981,7 @@
 	PC = pad_char[0];
 #endif /* OBSOLETE */
 
-#ifdef TAB3
+#if defined(TAB3) && defined(ONLCR) && defined(OCRNL) && defined(ONLRET)
     if (oldmode.c_oflag & (TAB3 | ONLCR | OCRNL | ONLRET)) {
 	oldmode.c_oflag &= (TAB3 | ONLCR | OCRNL | ONLRET);
 	SET_TTY(STDERR_FILENO, &oldmode);
